universal-login-react/src/ui/WalletSelector/Suggestions/InvalidForConnectionSuggestion.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const InvalidForConnectionSuggestion = () => (
+  <div className="hint">Name is invalid for connection</div>
```

Comment:
An account with this name does not exist.

Reply:
👍